### PR TITLE
extend list of extensions for R 3.2.1

### PR DESCRIPTION
**Pull request by boegel**
these extra extensions are required as deps for ShrinkBayes (PR coming up)

### CODE REVIEW

File: easybuild/easyconfigs/r/R/R-3.2.1-foss-2015b.eb [hunk 1]
```diff
@@ -366,8 +366,6 @@ exts_list = [
     ('SuperLearner', '2.0-15', ext_options),
     ('lpSolve', '5.6.13', ext_options),
     ('mediation', '4.4.5', ext_options),
-    ('lpSolve', '5.6.13', ext_options),
-    ('mediation', '4.4.5', ext_options),
```
Comment (wpoely86) [inline easybuild/easyconfigs/r/R/R-3.2.1-foss-2015b.eb]: gone cause?

Comment (boegel) [inline easybuild/easyconfigs/r/R/R-3.2.1-foss-2015b.eb]: duplicates, see lines above

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/10a262d6001d1122dd86 for a full test report.

**Comment by boegel** [comment]
going in, thanks for the review @wpoely86!

**Pull request merged**